src/reanimated2/animation/util.ts

```diff
@@ -43,6 +44,27 @@ if (__DEV__ && IS_REDUCED_MOTION) {
   );
 }
 
+const SHOULD_BE_USE_WEB = shouldBeUseWeb();
+
+export function assertEasingIsWorklet(
+  easing: EasingFunction | EasingFunctionFactory
+) {
+  'worklet';
+  if (_WORKLET) {
+    // If 'assertEasingIsWorklet' is called on UI runtime then easing is a copy and it is not a worklet
+    return true;
+  }
```

Comment:
We don't plan to call `assertEasingIsWorklet` on the UI runtime. This function is meant to work only on the RN runtime.

1. Let's remove `'worklet';` directive to forbid calling it from the UI runtime
2. Let's remove `if (_WORKLET) { ... }` since we don't need to support that
```suggestion
  'worklet';
  if (_WORKLET) {
    // If 'assertEasingIsWorklet' is called on UI runtime then easing is a copy and it is not a worklet
    return true;
  }
```

Reply:
I've added assertions each time `assertEasingIsWorklet` is called. Maybe we should move both `__DEV__` and `!_WORKLET` inside the fnciton?
```
 if (__DEV__ && userConfig?.easing && !_WORKLET) {
    assertEasingIsWorklet(userConfig?.easing);
  }
```

Reply:
No, `assertEasingIsWorklet` should only be called on the RN runtime. If it is called on the UI runtime, this means something went wrong and we should fix the root cause. There's no need to check `!_WORKLET`, please revert that change.